src/blueprints/tv2/value-objects/tv2-layers.ts

```diff
@@ -21,6 +21,8 @@ export enum Tv2SourceLayer {
   REPLAY_STUDIO_AUXILIARY = 'studio0_aux_studio_screen',
   REPLAY_VIZ_AUXILIARY = 'studio0_aux_viz_full1',
   VIDEO_CLIP = 'studio0_selected_clip',
+
+  ROBOT_CAMERA = 'studio0_robot_camera'
```

Comment:
I don't think there is a logical reason for that new line at the top. Enum values are logically related

Reply:
Updated